odoo/sql_db.py

```diff
@@ -28,6 +29,8 @@
 psycopg2.extensions.register_type(psycopg2.extensions.UNICODE)
 
 _logger = logging.getLogger(__name__)
+_logger_conn = logging.getLogger(__name__ + ".connection")
```

Comment:
```suggestion
_logger_conn = _logger.getChild("connection")
```

Having a child logger is useful if you wish to be able to turn on connection logging without the full SQL debugging. Then for consistency, all the logs related to establishing, closing and managing connections should go through that same child logger (regardless of the log level). The `_debug()` function of the ConnectionPool should use it too.

Reply:
Mister Olivier thanks for you review

It is an honor for me

I just attended all your feedback

Thank you!